content/en/docs/concepts/services-networking/service.md

```diff
@@ -1047,7 +1045,7 @@ spec:
 
 Using the userspace proxy for VIPs, work at small to medium scale, but will
 not scale to very large clusters with thousands of Services.  The [original
-design proposal for portals](http://issue.k8s.io/1107) has more details on
+design proposal for portals](https://issue.k8s.io/1107) has more details on
```

Comment:
Is it better to link to https://github.com/kubernetes/kubernetes/issues/1107?

(If not, maybe we should change all the other issue links to use issue.k8s.io - could be a separate PR of course).

Reply:
suggestion taken.

Reply:
btw, this shortened url does look nice, when compared to `https://github.com/kubernetes/kubernetes/issues/1107`.